### ISSUE TITLE
Add warm-up support before all test cases

### ISSUE THREAD

**Issue by dumarazvan** [drazvan91/YonTech.Fat#27]

**Comment by drazvan91** [comment]
This is a very good idea especially for the cases where the webservers take more than 20 seconds to start and throws timeouts for initial tests.

What do you think about this mechanism @dumarazvan ?

``` csharp
public class MyWarmup: FatWarmup
{
    // pages, sections, flows can be added and used
    GithubHomePage homePage { get; set; }

    public MyWarmup()
    { 
        // override the default timeout for warmup. 
        Timeout = 60;
    }
    
    // this is a method which warmups the server and returns true if the server is ready
    protected override bool Warmup()
    {
         WebBrowser.Navigate("https://github.com/");
         return homePage.LoginButton.Exists();
    }
}
```

**Comment by drazvan91** [comment]
Implemented in 0.8.1
